resources/views/home/download.blade.php

```diff
@@ -9,102 +9,116 @@
+                    </div>
+                    <a
+                        class="btn-osu-big btn-osu-big--download"
+                        href="{{ $lazerUrl }}"
+                    >
+                        <div class="btn-osu-big__content">
+                            <div class="btn-osu-big__left">
+                                {{ osu_trans('home.download.download') }}
+                                <div>
+                                    <div class="btn-osu-big__text-top btn-osu-big__text-top--download">
+                                        osu!
+                                    </div>
+                                    {{ osu_trans('home.download.for_os', ['os' => $lazerPlatformName]) }}
+                                </div>
+                                <div class="btn-osu-big__text-bottom btn-osu-big__text-bottom--download-version">
+                                    {{ $version }}
+                                </div>
+                            </div>
+                            <span class='btn-osu-big__icon'>
+                                <span class='fas fa-fw fa-download'></span>
+                            </span>
+                        </div>
                     </a>
-
-                    <div class="download-page-header__extra-links">
-                        <a class="download-page-header__extra-link" href="{{ osu_url('installer-mirror') }}">
-                            {{ osu_trans('home.download.mirror') }}
-                        </a>
-                        <span class="download-page-header__extra-link download-page-header__extra-link--separator"></span>
-                        <a class="download-page-header__extra-link" href="{{ osu_url('osx') }}">
-                            {{ osu_trans('home.download.macos-fallback') }}
-                        </a>
+                    <div class="download-page__other-platforms">
+                        @include('objects._basic_select_options', ['modifiers' => 'download', 'selectOptions' => $selectOptions])
                     </div>
                 </div>
             </div>
-            <div class="download-page-header__separator">{{ osu_trans('home.download.or') }}</div>
-            <div class="download-page-header__choice">
-                <div class="download-page-header__action-description">
-                    <h3 class="download-page-header__action-title">
-                        {{ osu_trans('home.download.action_lazer_title') }}
-                    </h3>
-
-                    <div>
+            <div class="download-page__banner-tail">
+                <div class="download-page__banner-content download-page__banner-content--tail">
+                    <div class="download-page__text download-page__text--download-stable">
                         <div>
-                            {{ osu_trans('home.download.action_lazer_description') }}
+                            {{ osu_trans('home.download.stable_text') }}
```

Comment:
the wordings and the lack of punctuation makes it read like one sentence

![](https://s.kohaku.love/2025/11/firefox_2025-11-11_15-46-48.png)